rawpointer_test.go

```diff
@@ -232,3 +232,24 @@ func TestRawPointerTotalListSize(t *testing.T) {
 		}
 	}
 }
+
+func TestLandingPadNearPointer(t *testing.T) {
+	tests := []struct {
+		far     rawPointer
+		tag     rawPointer
+		landing rawPointer
+	}{
+		{0x08, 0x2000200000000, 0x2000200000000},
```

Comment:
I've defined `Address` to be a byte address, not a word address.  `rawFarPointer` is doing the conversion and bit-shifting at the same time by just lopping off the bottom three bits.